install-poetry.py

```diff
@@ -0,0 +1,810 @@
+"""
```

Comment:
The script itself works from a basic test and I do think this is the right direction, ie. move away from platform bundles.

Reply:
It will also help make plugins straightfoward, i.e. entrypoints, which is not easily doable with the current solution.